sockeye/test_utils.py

```diff
@@ -157,8 +203,12 @@ def tmp_digits_dataset(prefix: str,
                 generate_odd_even_factors(test_target_path, test_factor_path)
                 data['train_target_factors'].append(train_factor_path)
                 data['dev_target_factors'].append(dev_factor_path)
-                data['test_target_factors'].append(dev_factor_path)
+                data['test_target_factors'].append(test_factor_path)
```

Comment:
btw I fixed this bug from previous code (dev_factor_path was used instead of test_factor_path)

Reply:
good catch, thanks!